EthOn.rdf

```diff
@@ -1936,7 +1936,7 @@
     <owl:Class rdf:about="http://ethon.consensys.net/ExternalAccount">
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/Account"/>
         <suggestedStringRepresentation xml:lang="en">ExternalAccount</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">An Account owned by an External Actor.</rdfs:comment>
+        <rdfs:comment xml:lang="en">One of three types of accounts on Ethereum, an External Account is an account owned by an External Actor. The majority of Ethereum accounts fall under this term.</rdfs:comment>
```

Comment:
@chronaeon 
Probably only in private blockchains Contract Accounts could exceed the number of External Accounts.

My comment was about making statements that are blockchain specific; it made me uncomfortable :crab: